components/Post.php

```diff
@@ -81,8 +81,12 @@ protected function loadPost()
          * Add a "url" helper attribute for linking to each category
          */
         if ($post && $post->categories->count()) {
-            $post->categories->each(function($category) {
-                $category->setUrl($this->categoryPage, $this->controller);
+            $blogPostsComponent = $this->getComponent('blogPosts', $this->categoryPage);
+
+            $post->categories->each(function($category) use ($blogPostsComponent) {
```

Comment:
@GinoPane space between `function` and arguments as per PSR-2 guidelines.
```suggestion
            $post->categories->each(function ($category) use ($blogPostsComponent) {
```